mod/wikirate/set/right/my_contributions.rb

```diff
@@ -48,14 +54,18 @@ def sections
 
   view :contribution_counts do |args|
     content_tag :div, :class=>'counts' do
-      card.sections.map do |count, name, contr_name|
-        content_tag :a, :class=>"item", :href=>"##{contr_name}" do
-          %{
-            <span class="#{name.downcase}">#{count}</span>
-            <p class="legend">#{name}</p>
-          }.html_safe
-        end
-      end.join("\n")
+      if card.sections 
+        card.sections.map do |count, name, contr_name|
+          content_tag :a, :class=>"item", :href=>"##{contr_name}" do
+            %{
+              <span class="#{name.downcase}">#{count}</span>
+              <p class="legend">#{name}</p>
+            }.html_safe
+          end
+        end.join("\n")
+      else
+        ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.